.github/workflows/lint.yml

```diff
@@ -0,0 +1,32 @@
+name: Linting Check for radis-app
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  run-linters:
+    name: Run linters
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      - name: Install Node.js dependencies
+        run: npm ci
```

Comment:
Would use `yarn install --frozen-lockfile` instead of npm here, such as the way it is for the main branch workflow: https://github.com/suzil/radis-app/blob/main/.github/workflows/main.yml#L33